templates/components/api-server/template/server/server.js

```diff
@@ -3,19 +3,21 @@ var boot = require('loopback-boot');
 
 var app = module.exports = loopback();
 
-// Bootstrap the application, configure models, datasources and middleware.
-// Sub-apps like REST API are mounted via boot scripts.
-boot(app, __dirname);
-
 app.start = function() {
   // start the web server
   return app.listen(function() {
-    app.emit('started');
```

Comment:
We must preserve the event `started`, it's used by [server/boot/explorer.js](https://github.com/strongloop/loopback-workspace/blob/f031f73f00b903204a674fcfdb6570fa0bad88cc/templates/components/api-server/template/server/boot/explorer.js#L20-L26) to print a message with Explorer URL, but only after the app was started.